contracts/token/ERC777/ERC777.sol

```diff
@@ -115,21 +115,21 @@ contract ERC777 is Context, IERC777, IERC20 {
      *
      * This implementation always returns `1`.
      */
-    function granularity() public view override returns (uint256) {
+    function granularity() public view virtual override returns (uint256) {
```

Comment:
I think this is okay because there needs to be a way to implement tokens with different granularity, but those contracts should also at the same time override transfer to make sure it reverts if the amount is not a multiple, so this should be documented.

Alternatively we add that check ourselves but that will cost gas, I don't think the compiler will optimize it.